src/scenes/ObservationsRouter/biomass/useExportBiomassDetailsZip.ts

```diff
@@ -2,37 +2,42 @@ import { useCallback } from 'react';
 
 import sanitize from 'sanitize-filename';
 
+import { selectPlantingSite } from 'src/redux/features/tracking/trackingSelectors';
+import { useAppSelector } from 'src/redux/store';
 import ObservationsService from 'src/services/ObservationsService';
 import strings from 'src/strings';
-import { ObservationResultsPayload } from 'src/types/Observations';
-import { PlantingSite } from 'src/types/Tracking';
 import downloadZipFile from 'src/utils/downloadZipFile';
 
 export default function useExportBiomassDetailsZip(
-  observation?: ObservationResultsPayload,
-  plantingSite?: PlantingSite
+  observationId?: number,
+  plantingSiteId?: number,
+  startDate?: string
 ) {
+  const plantingSite = useAppSelector((state) =>
+    plantingSiteId ? selectPlantingSite(state, plantingSiteId) : undefined
+  );
```

Comment:
I am not at all confident this is the right way to do this. The problem I was trying to solve was that the rows that are passed to the table view just have the summary data, which doesn't include a planting site name, and we need the planting site name to construct the exported filenames. We do have the planting site ID in the row data, so I'm using that to get the site name. The ternary expression here feels wrong to me but I'm not sure what the right alternative is (if any; maybe this is fine?)